Tribler/Core/Modules/restapi/rest_manager.py

```diff
@@ -30,7 +30,7 @@ def start(self):
         self.root_endpoint = RootEndpoint(self.session)
         site = server.Site(resource=self.root_endpoint)
         site.requestFactory = RESTRequest
-        self.site = reactor.listenTCP(self.session.get_http_api_port(), site)
+        self.site = reactor.listenTCP(self.session.get_http_api_port(), site, interface="127.0.0.1")
```

Comment:
Could this be implemented in a more flexible way? A command line option (`twistd -n tribler --bind-address=0.0.0.0`) to bind to (a specific) WAN IP would be nice.